Sample apps update (#888)
Macrometacorp/docs: docs/apps/graph-ql-caching.md

---
sidebar_position: 3

> title: GraphQl Caching

Reviewer: ```suggestion
title: GraphQL Caching
```

> title: GraphQl Caching

Reviewer: Will our users know what GraphQL is? I don't think it's an industry term.

---

> ## Macrometa GDN

Reviewer: I really hate having the tenant information in our docs. What value does it provide to our end user?

> | **Platform**                       | **Tenat**                      |

Reviewer: ```suggestion
| **Platform**                       | **Tenant**                      |
```

| ---------------------------------- | ------------------------------ |
| [URL](https://cache.eng.macrometa.io/) | `demo@macrometa.com` |

## Deployment

| **Source Code**                       | **Deployment**                      | 
| ---------------------------------- | ------------------------------ | 

> | [Github](https://github.com/Macrometacorp/parsec-graphql-cache) | [Deployment](https://macrometacorp.github.io/parsec-graphql-cache/) | 

Reviewer: This GitHub repo is private. This is not useful for our users.

> | [Github](https://github.com/Macrometacorp/parsec-graphql-cache) | [Deployment](https://macrometacorp.github.io/parsec-graphql-cache/) | 

Reviewer: ```suggestion
| [GitHub](https://github.com/Macrometacorp/parsec-graphql-cache) | [Deployment](https://macrometacorp.github.io/parsec-graphql-cache/) | 
```

> ## Solution

Reviewer: ```suggestion
## Solution

```

> Immerse yourself in a comprehensive demonstration that illuminates the tremendous strides made in optimizing GraphQL query response times while effectively alleviating the burden on the GraphQL origin server. Our demo showcases a significant architectural enhancement achieved by customizing the Quell Library (https://github.com/open-source-labs/Quell) to leverage GDN KV collection as a superior caching mechanism, surpassing traditional Redis server implementations. By meticulously configuring a robust 4-region US federation with GDN and deploying a single Elasticsearch instance strategically located in the US West region, we unlock unparalleled performance gains. Prepare to delve deep into the intricacies of this cutting-edge setup, unraveling the inherent power of GDN in revolutionizing GraphQL workflows.

Reviewer 2: @dlozina-macrometa can we please remove setup details from solution text?
```
Remove:
By meticulously configuring a robust 4-region US federation with GDN and deploying a single Elasticsearch instance strategically located in the US West region,
```

> Immerse yourself in a comprehensive demonstration that illuminates the tremendous strides made in optimizing GraphQL query response times while effectively alleviating the burden on the GraphQL origin server. Our demo showcases a significant architectural enhancement achieved by customizing the Quell Library (https://github.com/open-source-labs/Quell) to leverage GDN KV collection as a superior caching mechanism, surpassing traditional Redis server implementations. By meticulously configuring a robust 4-region US federation with GDN and deploying a single Elasticsearch instance strategically located in the US West region, we unlock unparalleled performance gains. Prepare to delve deep into the intricacies of this cutting-edge setup, unraveling the inherent power of GDN in revolutionizing GraphQL workflows.

Reviewer: ```suggestion
This sample application demonstrates how to optimize GraphQL query response times while reducing the burden on the GraphQL origin server. This demo includes a customized Quell Library (https://github.com/open-source-labs/Quell) to leverage a Macrometa key-value collection as a caching mechanism, surpassing traditional Redis server implementations. We achieve performance gains by configuring a four-region US federation with GDN and deploying a single Elasticsearch instance located in the US West region. 
```

Reviewer: Wrong tone for technical documentation. It sounds like you swallowed a salesman.
